wallet/wallet/errors.go

```diff
@@ -6,17 +6,17 @@ import (
 )
 
 var (
-	ErrIsolatedWalletCantGenerateKeyPairs = errors.New("isolated wallet can't generate key pairs")
-	ErrIsolatedWalletDoesNotHaveMasterKey = errors.New("isolated wallet doesn't have a master key")
-	ErrCantRotateKeyInIsolatedWallet      = errors.New("isolated wallet can't rotate key")
-	ErrInvalidRecoveryPhrase              = errors.New("recovery phrase is not valid")
-	ErrPubKeyAlreadyTainted               = errors.New("public key is already tainted")
-	ErrPubKeyIsTainted                    = errors.New("public key is tainted")
-	ErrPubKeyNotTainted                   = errors.New("public key is not tainted")
-	ErrPubKeyDoesNotExist                 = errors.New("public key does not exist")
+	ErrIsolatedWalletCantGenerateKeys     = errors.New("an isolated wallet can't generate keys")
```

Comment:
```suggestion
	ErrIsolatedWalletCannotGenerateKeys     = errors.New("an isolated wallet cannot generate keys")
```

Same principe as `ErrWalletDoesNotExists` - we use `DoesNot` instead of `Dont`

Reply:
Same as above.